src/App.jsx

```diff
@@ -1,8 +1,81 @@
-import logo from "./logo.svg";
 import "./App.css";
+import { useState, useEffect } from "react";
 
 function App() {
-    return <div className="bg-blue-300 w-screen h-screen"></div>;
+    const [meme, setMeme] = useState({
```

Comment:
May be you can use different states because it will help you to updated there states efficency.